blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+
+  This fix is a crucial one in this release. Many users reported non-working
+  on-board displays with different CPUs than those which were tested in 3mdeb
+  lab. Now all Alder Lake-S graphics devices should be initialized properly and
+  give firmware output on the screen.
+
+- [MSI PRO Z690-A WIFI DDR4 with two Video Cards (2x Radeon 5600XT) has issues related to MMIO resource allocation](https://github.com/Dasharo/dasharo-issues/issues/245)
+
+  Quite specific issue with board configuration overloaded with GPU cards in
+  the community. As such configuration is not common, the debugging involved
+  the issue reporter cooperation by flashing custom binaries and providing
+  debug logs from Dasharo firmware. It happened that the memory requested by
+  both of the cards was too big to fit into the 32 bit space. The fix for this
+  one required reserving much more space for devices' memory in the 32 bit
+  space, by shifting some of the usable RAM in 32 bit space to 64 bit space.
+  Such change let the firmware allocate all memory resources required by both
+  GPUs.
+
+- [Suspend doesn't work in Qubes OS with v1.1.0](https://github.com/Dasharo/dasharo-issues/issues/271)
+
+  Another issue discovered by community. One of Dasharo Team changes introduced
+  an regression in the suspend/resume feature. All OSes were affected. Fix has
+  been applied and the suspend/resume should now work on all OSes. The tests
+  have been also extended with multiple suspend/resume cycles to avoid such
+  problem in the future.
+
+- [Intel XTU on Windows reports "The platform does not support overclocking" on the MSI PRO Z690-A WIFI DDR4 with a K-series Processor](https://github.com/Dasharo/dasharo-issues/issues/159)
+
+  The issue has been reported by community in the first days after v1.0.0
+  release. Surprisingly the Intel XTU utility reported that platform does not
+  support overclocking, despite all components had all the overclocking
+  capabilities. The culprit was the OC lock bit being set by default in FSP
+  which prevented Intel XTU from modifying any settings from Windows level, so
+  it reported the platform as unsupported. The OC lock changes the
+  initialization path inside FSP, so you may encounter some unexpected changes
+  in CPU configuration. Please do not hesitate to report them. One of such
+  unexpected changes was an override of CPU turbo limits and causing the CPU
+  frequency to be lower than maximum default value (and so the performance was
+  degraded).
+
+- [SATA ports malfunction or Hot-Plug function disabled](https://github.com/Dasharo/dasharo-issues/issues/315)
+
+  A feature request from community to enable SATA hotplug functionality.
+
+- Platform sometimes automatically powers on after power off
+
+  Noticed by one of Dasharo Team developers when dGPUs are plugged. It happened
+  that the dGPUs were sending ACPi SCI interrupts which are not currently
```

Comment:
```suggestion
  that the dGPUs were sending ACPI SCI interrupts which are not currently
```

Reply:
https://github.com/3mdeb/news-and-ideas/pull/513/commits/96283809e0eeabcfe1c7d410854896114d3588c8